website_odoo_debranding/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+#. Go to your home page.
+#. Enable or disable *Customize > Footer Copyright > Remove Odoo Promotional
```

Comment:
I don't find the "Remove Odoo Promotional" link. Could you check?

Reply:
Yes you are right, this is not necessary, Done! :+1: